blog.html

```diff
@@ -0,0 +1,100 @@
+         <div class="blog-container-box">
+            <img src="images/blog-2.jpg" alt="">
+         </div>
+         <div class="blog-container-box">
+            <h3>Traveling With Your Dog</h3>
+            Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga.
+            Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit
+            ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam
+            sus am aut amet ant fugiatum, utem non reptat.
+            Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur,
+            con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad
+            qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim
+            quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.
+         </div>
+      </div>
+      <div class="blog-container 3">
+         <div class="blog-container-box">
+            <img src="images/blog-3.jpg" alt="">
+         </div>
+         <div class="blog-container-box">
+            <h3>Traveling With Your Dog</h3>
+            Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga.
+            Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit
+            ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam
+            sus am aut amet ant fugiatum, utem non reptat.
+            Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur,
+            con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad
+            qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim
+            quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.
+         </div>
+      </div>
+      <div class="">
+         <a href="#">  See More Dogs</a>
+      </div>
+      <footer>
+         <div class="footer-container">
+            <div class="footer-image-container">
+               <img src="images/adoptadog-logo.svg" alt="">
+            </div>
+            <div class="">
+              <p> Aborpos nisto inci tet aut que volupti beruptiberro to ex <br>
+              esseque invenienihil imaximu scideru ptistiusam consequunt</p>
+            </div>
+            <div class="">
+              <ul>
+                 <li><a href="index.html">Home / </a></li>
+                 <li> <a href="dogs.html">Dogs / </a></li>
+                 <li><a href="blog.html">Blog </a></li>
+              </ul>
+            </div>
+            <div class="">
+              © 2018 Adoptadog, LLC | Seattle, WA
```

Comment:
You can handle the copyright with a  `&copy;` element

also, I'm not a huge fan of just text in a div, instead maybe just a p tag or something. To use the copy element it would look like this...

`<p>&copy 2018 Adoptadog, LLC | Seattle, WA</p>`